xml/Microsoft.Extensions.Logging/LoggingBuilderExtensions.xml

```diff
@@ -140,8 +140,8 @@
       </AssemblyInfo>
       <ReturnValue />
       <Docs>
```

Comment:
@joelmartinez @TianqiZhang @mimisasouvanh this method takes two parameters, but they were not included in the `<Docs>` section. Seems like a bug.